### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by krichardson864**
We should attempt to merge the current version of Develop with Master before we try to split the code

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -34,35 +35,37 @@ def index():
 	to see more.
 	""")
 
-def newPosts(sr):
-	return reddit.subreddit(sr).new(limit=100)
+def newPosts(searchbase, searchbase2 = None):
```
Comment (Brojowski) [inline app.py]:
```suggestion
def newPosts(searchbase):
```

File: app.py [hunk 2]
```diff
@@ -34,35 +35,37 @@ def index():
...
+	return searchbase.new(limit=100)
 
-def hotPosts(sr):
-	return reddit.subreddit(sr).hot(limit=100)
+def hotPosts(searchbase, searchbase2 = None):
```
Comment (Brojowski) [inline app.py]:
```suggestion
def hotPosts(searchbase):
```

File: app.py [hunk 3]
```diff
@@ -34,35 +35,37 @@ def index():
...
+	return searchbase.hot(limit=100)
 
-def topPostsAllTime(sr):
-	return reddit.subreddit(sr).top(time_filter='all', limit=100)
+def topPostsAllTime(searchbase, searchbase2 = None):
```
Comment (Brojowski) [inline app.py]:
```suggestion
def topPostsAllTime(searchbase):
```

File: app.py [hunk 4]
```diff
@@ -34,35 +35,37 @@ def index():
...
+	return searchbase.top(time_filter='all', limit=100)
 	
-def topPostsPast24Hours(sr):
-	return reddit.subreddit(sr).top(time_filter='day', limit=100)	
+def topPostsPast24Hours(searchbase, searchbase2 = None):
```
Comment (Brojowski) [inline app.py]:
```suggestion
def topPostsPast24Hours(searchbase):
```

File: app.py [hunk 5]
```diff
@@ -34,35 +35,37 @@ def index():
...
+	return searchbase.top(time_filter='day', limit=100)
 
-def controversalPostsAllTime(sr):
-	return reddit.subreddit(sr).controversial(time_filter = 'all', limit=100)
+def controversalPostsAllTime(searchbase, searchbase2 = None):
```
Comment (Brojowski) [inline app.py]:
```suggestion
def controversalPostsAllTime(searchbase):
```

File: app.py [hunk 6]
```diff
@@ -34,35 +35,37 @@ def index():
...
+	return searchbase.controversial(time_filter='all', limit=100)
 	
-def controversalPast24Hours(sr):
-	return reddit.subreddit(sr).controversial(time_filter = 'day', limit=100)
+def controversalPast24Hours(searchbase, searchbase2 = None):
```
Comment (Brojowski) [inline app.py]:
```suggestion
def controversalPast24Hours(searchbase):
```

File: testing.py [hunk 7]
```diff
@@ -0,0 +1,78 @@
+import praw
+
+class RedditConfig:
+    id = '3iaAttC7v0D96Q'
+    secret = 'Dm-EfQSVyl8skVenes3DH25QfkY'
```
Comment (42B) [inline testing.py]: Should these credentials be on GitHub?

Comment (Brojowski) [reply testing.py]: Hi @42B,

Thanks for your interest in our project. You are absolutely correct that these credentials should not be on GitHub, thank you for pointing that out. I have removed the keys so they should no longer be valid and will generate new ones which we will handle better.

Comment (Brojowski) [reply testing.py]: Also @42B, could I ask how you found our project?

Comment (42B) [reply testing.py]: @Brojowski 

Sure, I go to this [url](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+language%3Apython+) and look at new pull requests in Python for practice.

**Pull request merged**